apps/website/src/app/components/Footer.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { TwitterIcon } from "@/app/components/icons/TwitterIcon";
+import { GithubHoverableLogo } from "@/app/components/icons/GithubHoverableLogo";
+
+export const Footer = () => {
+  return (
+    <footer className="z-10 bg-white relative bot-0 w-full h-fit flex flex-col gap-3 py-5 items-center self-stretch gt_mobile:flex-row gt_mobile:justify-between gt_mobile:py-6 gt_mobile:px-28 gt_mobile:self-stretch border-t border-gray-200">
+      <p className="not-italic font-normal text-gray-500 text-sm gt_mobile:text-footer_text gt_mobile:font-light">
+        © NameHash Labs. All Rights Reserved
```

Comment:
```suggestion
        &copy; NameHash Labs. All Rights Reserved
```

Reply:
Will do that in other PR today, just to not forget about it.